Functions/xASL_io_ReadDataPar.m

```diff
@@ -0,0 +1,250 @@
+function [x] = xASL_io_ReadDataPar(DataParFile)
+% xASL_io_ReadDataPar This function reads DATA_PAR .json or .m file and creates the x structure.
```

Comment:
`DATA_PAR` is a meaningful term to us, but not to other first-time code readers. We don't need to keep calling it DataPar, but at least let's add some explanation:

"This deals with reading/parsing the data parameter file, which is a file containing settings specific to processing a certain dataset or study. Was historically abbreviated to DataPar".

Reply:
We use the same name in the master script though ;) so I think that's okay. Alternatively we can link to that

Reply:
Improved the explanation.